winch/Cargo.toml

```diff
@@ -20,6 +20,8 @@ anyhow = { workspace = true }
 wasmparser = { workspace = true }
 clap = { workspace = true }
 wat = { workspace = true }
+cranelift-codegen = { workspace = true }
+capstone = "0.9.0"
```

Comment:
Could this dep be defined at the workspace level (and shared with clif-util) similar to the others above?

Reply:
I considered it, but didn't because clif-util [declares capstone as optional](https://github.com/bytecodealliance/wasmtime/blob/main/cranelift/Cargo.toml#L39), bound to the `disas` feature, and as far as I know workspace dependencies can't be optional.

Reply:
Ah, right, that's fine then.

Reply:
Actually upon verification it works fine, my bad. I confused using a workspace dependency as optional vs _declaring_ a workspace dependency as optional; and the latter is what doesn't work.